tests/index.js

```diff
@@ -36,6 +35,7 @@ describe('<ContentLoader />:', () => {
         speed={10}
         height={200}
         width={200}
+        animate={true}
```

Comment:
The animate is already true then you don't need to set again

Reply:
Changed to false to test